internal-packages/next-dev/README.md

```diff
@@ -58,6 +58,9 @@ if (process.env.NODE_ENV === 'development') {
 }
 ```
 
+
+> View [this file](https://github.com/cloudflare/next-on-pages/blob/main/internal-packages/next-dev/src/index.ts#L31-L60) for a list of available binding types.
```

Comment:
```suggestion
> View [these types](https://github.com/cloudflare/next-on-pages/blob/main/internal-packages/next-dev/src/index.ts#L31-L60) for a list of available binding types.
```

or something along those lines as user should not care about the whole file no? 🤔